ocs_ci/ocs/cluster_load.py

```diff
@@ -313,6 +314,7 @@ def reach_cluster_load_percentage(self):
         logger.info(wrap_msg(msg))
         self.target_pods_number = len(self.dc_objs)
 
+    @retry(IndexError, tries=5, delay=5, backoff=1)
```

Comment:
is 5 seconds delay enough?  I think we can even try more like 20 seconds.

Reply:
overall it's 25 seconds that we wait for the queries to work. It's hard to say (since we hit this only twice without and successful RCA), but that should be enough IMO